NOTICE

```diff
@@ -24,43 +45,31 @@ The derived files are:
 - baremaps-core/src/main/proto/vector_tile.proto
 
 This product includes code derived from Palantir Streams.
-Apache License, Version 2.0
+Apache License 2.0.
 Please visit the following URL for the full text of the Palantir license:
 https://github.com/palantir/streams/blob/develop/LICENSE
 The derived files are:
 - baremaps-core/src/main/java/org/apache/baremaps/stream/BufferedSpliterator.java
 
 This product includes code derived from Planetiler.
-Apache License, Version 2.0
+Apache License 2.0.
 Please visit the following URL for the full text of the Planetiler license:
 https://github.com/onthegomap/planetiler/blob/main/LICENSE
```

Comment:
If a dependency uses the Apache License 2.0 (AL2) and includes a NOTICE file, we need to add the corresponding NOTICE file here.

https://github.com/onthegomap/planetiler/blob/main/NOTICE

Reply:
Notice that we don't include planetiler which has GPL dependencies, we only integrated a data structure. Therefore, I'm tempted to say that we don't need to include the content of their NOTICE file.